webpack/common.config.js

```diff
@@ -158,6 +168,7 @@ module.exports = {
         ]
         // delete: [path.resolve(__dirname, '..', 'dist-build')] // will cause errors when watching files
       }
-    })
+    }),
+    new i18nGenerator(`./dist-build/locales/{en,de}.json`, './dist/static/languages/')
```

Comment:
similar idea as mentioned in another comment:
how about extracting the value languages to a `cons and move it to the top, so it's more exposed and easier to find once a new language is supported?
here, you could then
```js
  `./dist-build/locales/{${['en', 'de'].join(',')}}.json`     // note: i don't know if this works right away (syntax)
```
here